wiki/Tournaments/UKCC/UKCC8/en.md

```diff
@@ -0,0 +1,163 @@
+- Seeding for groups will be determined from your seed based on your osu! rank (as of signup close) and your overall position in the RO64 qualifer, eg 3rd seed + 15th in qualifier = 18 points
+
+### Rolling
+
+- When both players are in the lobby and ready, the referee will ask the players to !roll. The order of picks of bans is decided using the [UKCC Roll System](https://i.imgur.com/S0VIvni.png).
+
+### Banning
+
+- From the Group Stage onwards players will have **TWO** bans.
+- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+
+### Warmups
+
+- There will be no warmups in Qualification rounds. After this, warmup maps must be a submitted beatmap, and have a drain time of 4 minutes or less
+
+### Picking
+
+- Players have **2 minutes** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- For Freemod picks, players must select any combination of the following mods: **Hidden, HardRock**
+- Failed scores WILL count
+- Tiebreakers are Freemod, however No Mod can be used.
+
+## Tournament Schedule
+
+![Schedules](img/schedules.png)
+
+| Event | Timestamp |
+| --: | :--|
+| Signups Open | 22nd February 2020 |
+| Signups Closed | 15th March 2020 |
+| Group Stage | 28th/29th March 2020 |
+| Round of 32 | 4th/5th April 2020 |
+| Round of 16 + Losers Round 1 | 11th/12th April 2020 |
+| Quarter Finals + Losers Round 2/3 | 18th/19th April 2020 |
+| Semi Finals + Losers Round 4/5 | 25th/26th April 2020 |
+| Winners Finals + Losers 6/Semi Final | 2nd/3rd May 2020 |
+| Losers Final + GRAND Finals | 9th/10th May 2020 |
+
+## Prizes
+
+![Prizes](img/prizes.png)
+
+| Placing | Prize(s)|
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "Winner") | tba |
+| ![Silver crown](/wiki/shared/crown-silver.png "Second place") | tba |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "Third place") | tba |
+
+## Organisation
+
+![Staff List](img/staff.png)
+
+UK is run by many wonderful people, this list doesn't fully represent how many amazing people contribute as we accept people along the way. If you're interested in helping staff future UK Community Cups, please contact a host in the [Discord](https://discord.gg/EtsU7Cw)!
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host(s) | ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972)  |
+| Referees | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), tba |
+| Commentators | tba |
+| Streamers | ![][flag_GB] [Bae-](https://osu.ppy.sh/users/6576972), tba |
+| Mappoolers | tba |
+| Statistics | ![][flag_NO] [Yokespai](https://osu.ppy.sh/users/6399568) |
+| Graphics | ![][flag_GB] [Melonturtle](https://osu.ppy.sh/users/4775283) |
+
+## Teams
+
+A list of teams will be released upon the completion of the signup/screening phase
```

Comment:
```suggestion
*A list of teams will be released upon the completion of the signup/screening phase*
```
Shake things up with italics

Reply:
While this visually works, it gives this lint error:
![image](https://user-images.githubusercontent.com/31848108/76165316-ea547100-614d-11ea-8b83-9474e65022e7.png)

Reply:
Give a full stop at the very end (?)